src/domain/courseSearch/Search.tsx

```diff
@@ -93,6 +96,7 @@ const Search: React.FC<Props> = () => {
     setSelectedPlaces(places);
     setSelectedTexts(text);
     setSelectedDateTypes(dateTypes);
+    setIsFree(isFree || false);
```

Comment:
This style caught my eye, nothing wrong with this implementation but

`setIsFree(!!isFree)` or `setIsFree(Boolean(isFree))` could also be used.

No need to change this, use the one that you think is best.

Also noticed that this is done differently in eventSearch/Search.tsx. Check that one out, do we need the state at all?

Reply:
yea, the implementations is a bit different in those 2. in courses its inside the hook, but not in events.. even though its the same functionality. i just didnt try to refactor, as it might be due to some business logic diffs which i dont see now.

Reply:
Ok, I will check when I'm back at computer

Reply:
sure

Reply:
Compare isFree to alsoOngoingCourses in this file. isFree could be handled the same way?